### PR TITLE
Strongly typed improvements

### PR DESCRIPTION
**Pull request by burtybob**

### CODE REVIEW

**Comment by kinbiko** [comment]
You've created a NonSceneScripts folder. Please use spaces in folder, and place inside Scripts, or alternatively, place both Scripts and NonSceneScripts inside a folder called Code.

**Comment by kinbiko** [comment]
SceneName and TagName should also be in NonSceneScripts, if I've understood it's purpose correctly

File: Assets/NonSceneScripts/Extensions.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public static class Extensions
+{
+    public static void LoadNextScene(this Scene scene)
+    {
+        SceneManager.LoadScene(scene.buildIndex + 1);
```
Comment (kinbiko) [inline Assets/NonSceneScripts/Extensions.cs]: !fix Use the facade

File: Assets/Scripts/LevelHandler.cs [hunk 2]
```diff
@@ -21,11 +18,11 @@ void Start()
 
     public void LoadNextLevel()
     {
-        SceneManager.LoadScene(SceneManager.GetActiveScene().buildIndex + 1);
+        SceneManager.GetActiveScene().LoadNextScene();
```
Comment (kinbiko) [inline Assets/Scripts/LevelHandler.cs]: Use facade

Comment (kinbiko) [inline Assets/Scripts/LevelHandler.cs]: !fix

Comment (burtybob) [inline Assets/Scripts/LevelHandler.cs]: !fixed

File: Assets/NonSceneScripts/Extensions.cs (hunk 1 above)
Comment (burtybob) [inline Assets/NonSceneScripts/Extensions.cs]: !fixed

**Pull request merged**